### PR TITLE
Table and updates

### PR DESCRIPTION
**Pull request by Drache93**
* Update deps
* Export table
* Add additional styling to table
* Fix typing errors
* Use flex-start and flex-end instead of start and end for Box horizontal align
* Fix imports for widget stories

### CODE REVIEW

File: packages/bootstrap/src/components/icons/CaretLeft.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Icon, IconProps } from './Icon';
+
+export const CaretLeft: React.FC<IconProps> = (props) => (
+  <Icon {...props} viewBox="0 0 192 512">
```
Comment (alex-melnik) [inline packages/bootstrap/src/components/icons/CaretLeft.tsx]: Can we set placement via props and then rotate icon according to this?

**Comment by alex-melnik** [comment]
@Drache93 
I think we have to show just presenters in widget stories (without data providers)

**Comment by Drache93** [comment]
@alex-melnik Would you or someone be able to update those stories? I was just trying to get them to not crash! :D

File: packages/bootstrap/src/components/icons/CaretLeft.tsx (hunk 1 above)
Comment (Drache93) [reply packages/bootstrap/src/components/icons/CaretLeft.tsx]: We could do, but we don't save much weight. I think the current way guarantees consistent quality better.

**Comment by alex-melnik** [comment]
> @alex-melnik Would you or someone be able to update those stories? I was just trying to get them to not crash! :D

sure thing! We are going to do this after all stuff regarding dashboard will be merged

**Comment by Drache93** [comment]
@alex-melnik Thanks! I'll pull this in for now then, so AID can be completed. Stories can be fixed in the dashboard branches.

**Pull request merged**